### ISSUE TITLE
[Figma File Search] ...

### ISSUE THREAD

**Issue by tancho-speka** [raycast/extensions#3889]
### Extension

https://github.com/raycast/extensions/tree/4a5d8a3e4a789fbfafeadc9a538d9be62477ee1f/extensions/figma-files/

### Description

Error: Rendering failed:
Missing required component property "title"
Reason: Code: 2 (Unhandled Extension Rejection)
Domain: commands
Time: 07:06:01.153

### Steps To Reproduce

Precondition: 
You need to have Figma

STR:
1. Open Reycast
2. Make hotkey for Figma
3. Try to open it

### Current Behaviour

Something went wrog

### Expected Behaviour

Everything correct

**Comment by pernielsentikaer** [comment]
Hi @tancho-speka 👋 

I just tried to reproduce this, and I'm unable to.

When you get the `Something went wrong`, can you see if you can copy the error?

**Comment by pernielsentikaer** [comment]
We just released a patch that might fix this.

Could you run `Check For Extensions Updates` and try again 😊

**Comment by pernielsentikaer** [comment]
@tancho-speka we could take a debug session in order to see if we can catch this. Could you book a [time here](https://cal.frontapp.com/raycast/per/book) for the debugging 😊

**Comment by git-mikko** [comment]
i had the same bug last week.
reinstalling the extension helped me.

**Comment by ys-chung** [comment]
I just encountered the same issue.

```
Error: Rendering failed:
Missing required component property "title"
```

**Comment by michaelwschultz** [comment]
I believe this was solved in #6673.

**Comment by michaelwschultz** [comment]
@raycastbot close this issue
